tests/knowledge_base/test_actions.py

```diff
@@ -98,12 +120,15 @@ def compare_slots(slot_list_1, slot_list_2):
             [
                 SlotSet(SLOT_MENTION, None),
                 SlotSet(SLOT_ATTRIBUTE, None),
-                SlotSet(SLOT_OBJECT_TYPE, "restaurant"),
+                SlotSet(SLOT_OBJECT_TYPE, None),
```

Comment:
Could we actually add in this unit test the assertion that before the reset of `SLOT_OBJECT_TYPE`, there is that `SlotSet` event for setting `SLOT_OBJECT_TYPE` to the expected value (i.e `restaurant`) temporarily during the running of the knowledge base action?

Reply:
Got it. We added the assertion in `test_action_run` below.